src/autorestSession.ts

```diff
@@ -12,9 +12,9 @@ export interface AutorestOptions {
   restLevelClient?: boolean;
   rlcShortcut?: boolean;
   azureArm?: boolean;
-  addCredentials?: boolean;
```

Comment:
I think we should keep these settings available for backward compatibility. If a swagger has security scopes and also these passed in the config, these should win over the swagger's